jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/comments/CaseCommentItemView.java

```diff
@@ -173,4 +201,79 @@ public void updateCommentPressingEnter(@ForEvent("keyup") final KeyboardEvent e)
         }
     }
 
+    @Override
+    public CaseCommentSummary getValue() {
+        return caseCommentSummary.getModel();
+    }
+
+    @Override
+    public void setValue(final CaseCommentSummary model) {
+        this.caseCommentSummary.setModel(model);
+        this.updateCommentText.setValue(model.getText());
+        setEditMode(editMode);
+    }
+
+    protected void updateActions(final String author,final String id, final boolean editItem){
+        actionsItems.setInnerHTML("");
+        if (identity.getIdentifier().equals(author)) {
+            addAction( new CaseCommentsPresenter.CaseCommentAction() {
+                @Override
+                public String label() {
+                    return translationService.format(DELETE);
+                }
+
+                @Override
+                public void execute() {
+                    presenter.deleteCaseComment(id);
```

Comment:
Could we pass the object model in here (CaseCommentSummary) instead of just the String id.